lib/movements.js

```diff
@@ -478,6 +528,11 @@ class Movements {
 
     if (this.getBlock(node, 0, 0, 0).liquid) return // cant jump from water
 
+    let cost = 1 // Mostly used for checking if entities are in the jump path. Will be useful incase we want to prefer a path with no potential hostiles that can knock us out of our jump
```

Comment:
Should cost not start at 0? Cost used to be defined like this `const cost = this.exclusionStep(blockB) + 1` before being added to the move. Now it would be one more.

Reply:
The initial cost should be 1 however, since I changed the function to use a shared cost variable, the '+ 1' will need to be removed from each exclusionStep()